src/eslint-config-adeira/src/presets/flowtype.js

```diff
@@ -27,7 +29,7 @@ module.exports = ({
     'ft-flow/no-duplicate-type-union-intersection-members': ERROR,
     'ft-flow/no-existential-type': ERROR, // https://github.com/facebook/flow/issues/6308
     'ft-flow/no-flow-fix-me-comments': OFF,
-    'ft-flow/no-flow-fix-me-in-strict-files': OFF, // TODO: enable (after fixing our codebase)
+    'ft-flow/no-flow-suppressions-in-strict-files': OFF, // TODO: enable (after fixing our codebase)
```

Comment:
BTW @Brianzchen - seems like you are about to release this breaking change in a patch version. 😬

Reply:
Thanks forgot about this one. Yes I'll have to release this under v2.0.0 then if that works for you